### ISSUE TITLE
VIM losing syntax in current line (osx)

### ISSUE THREAD

**Issue by weslleyaraujo** [baskerville/bubblegum#4]
Hello @baskerville,
At first awesome work on bubblegum.
I am having this issue that the current highlighted line is always losing the syntax colors...

Here's the details:

**VIM-7.4.1016 - Mac OSX:**

![screen shot 2016-02-15 at 12 56 34 pm](https://cloud.githubusercontent.com/assets/2286385/13048432/1b0bbc08-d3e6-11e5-9295-8ffaedc407ae.png)

**VIM-7.4.629 Linux:**
![screen shot 2016-02-15 at 12 56 17 pm](https://cloud.githubusercontent.com/assets/2286385/13048442/2fb8196c-d3e6-11e5-941f-8f242eebe91a.png)

and thats my current `~/.vimrc` (which is shared by both systems):

``` sh
" =======================================
" .vimrc file
" =======================================
"
" My default config and plugins for vim :)
" Weslley Araujo <http://github.com/weslleyaraujo>
" =======================================

" Load plugins
"
if filereadable(expand("~/vundle.vim"))
  source ~/vundle.vim
endif

" =======================================
" My default configs
" =======================================
syntax enable
set encoding=utf-8

" tab
set tabstop=2 
set shiftwidth=2
set softtabstop=2
set autoindent
set expandtab 

" interface
set wildmenu        " set vim menu
set ruler           " show cursor position
set cursorline      " hightlight for actual line
set number          " show numbers
set hlsearch        " set highlight for search
set nowrap          " set no wrap for big lines
set nobackup        " set no backup
set incsearch
set background=dark
set backspace=2     " fix backspace bug [dont really know why :( ]
hi link htmlLink NONE "remove underline from links in html pages

" =======================================
" Files syntax
" =======================================
autocmd BufNewFile,BufRead {*.html.ejs,*.erb} set syntax=html
autocmd BufNewFile,BufRead {Gemfile,Gemfile.lock,*.ru} set syntax=ruby
autocmd BufNewFile,BufRead *.json set syntax=javascript

" =======================================
" Maps
" =======================================
nmap <c-n> :bn<CR> " map (ctrl + n) => :bn // buffer next
nmap <c-p> :bp<CR> " map (ctrl + p) => :bp // buffer prev

" =======================================
" Colorscheme
" =======================================
try
    colorscheme bubblegum-256-dark
catch /^Vim\%((\a\+)\)\=:E185/
    echo 'Cant find colorscheme bubblegum yet!'
endtry
```

And also the file which is being loaded up there (`vundle.vim`) :

``` sh
" ========================================
" Vim plugin configuration
" ========================================
"
" This file contains the list of plugin installed using vundle plugin manager.
" Once you've updated the list of plugin, you can run vundle update by issuing
" the command :BundleInstall from within vim or directly invoking it from the
" command line with the following syntax:
" vim --noplugin -u vim/vundles.vim -N "+set hidden" "+syntax on" +BundleClean! +BundleInstall +qall
" Filetype off is required by vundle

filetype off

set rtp+=$HOME/.vim/bundle/vundle/
call vundle#begin("$HOME/.vim/bundle")

" let Vundle manage Vundle (required)

Bundle 'gmarik/vundle'

" Generic

Plugin 'mattn/emmet-vim'
Plugin 'editorconfig/editorconfig-vim'

" Syntax hightlighting & colors

Plugin 'baskerville/bubblegum'

call vundle#end()

"Filetype plugin indent on is required by vundle
filetype plugin indent on
```

Any ideas what can be causing it?

**Comment by baskerville** [comment]
This was fixed in 8f99075e6201ede0baca2be3cf98314e8f370f70.

So `:PluginUpdate` might be the answer.

**Comment by weslleyaraujo** [comment]
after updating it is working as expected, thanks! :+1:
